### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by wSedlacek**
# Description

As a Developer, I would like to have smaller webpack bundles, so that I don't run up my netlify build minutes and my users have smaller load times

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Reworked rendering test

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by wSedlacek** [comment]
[[Developer] - Clean webpack](https://trello.com/c/5beHijTB/46-developer-clean-webpack)

**Comment by wSedlacek** [comment]
Staging: https://5e308059994da9000a94a652--story-squad.netlify.com

**Review by EricBanker12 — APPROVED** [review]
All tests passed and manual testing worked.

File: src/app/components/pdf/display/display.component.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    React.useEffect(() => {
+        const canvas = canvasRef.current;
+        if (!canvas?.getContext) return;
+
+        document.getPage(page).then((pdfPage) => {
+            const viewport = pdfPage.getViewport({ scale });
+            const canvasContext = canvas.getContext('2d');
+            if (!canvasContext) return;
+
+            console.log(viewport);
```
Comment (EricBanker12) [inline src/app/components/pdf/display/display.component.tsx]: Should probably remove this `console.log` before merging

**Pull request merged**